docs/pages/web/tiledata.js

```diff
@@ -0,0 +1,148 @@
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+        - When grouping Datapoints that aren't selectable
+        - For selectable information that is not part of a data visualization
+      `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Always present one tile in its selected state on default."
+            defaultCode={`<TileData color="01" title="Impressions" value="2M" selected trend={{ value: 5, accessibilityLabel: 'Trending up' }} /> `}
+          />
+
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use TileData to present a single option, since TileData's don't need to be selected. Use [Datapoint](https://gestalt.pinterest.systems/web/datapoint) instead."
+            defaultCode={`<TileData color="01" title="Impressions" value="2M" trend={{ value: 1, accessibilityLabel: 'Trending up' }}/> `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use the `showCheckbox` property when multiple Tiledatas can be selected. See the [group](https://gestalt.pinterest.systems/web/tiledata#Group) variant."
+            defaultCode={`<Flex gap={2}> <TileData color="01" title="Impressions" selected value="2M" showCheckbox trend={{ value: 1, accessibilityLabel: 'Trending up' }}/> <TileData color="02" title="Impressions" selected showCheckbox value="2M" trend={{ value: 1, accessibilityLabel: 'Trending up' }}/>   </Flex>`}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use Tiledata when multiple items can be selected without a visible checkbox."
+            defaultCode={`<Flex gap={2}> <TileData color="01" title="Impressions" selected value="2M"  trend={{ value: 1, accessibilityLabel: 'Trending up' }}/> <TileData color="02" title="MAU" selected  value="2M" trend={{ value: 1, accessibilityLabel: 'Trending up' }}/>   </Flex>`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <AccessibilitySection
+        name={generatedDocGen?.displayName}
+        description="Users should be able to navigate or activate TileData using a keyboard or other input modalities. Be sure to include an `accessibilityLabel` for the screen reader if you're using the `trend` or `tooltip` label."
+      />
+
+      <MainSection
+        name="Localization"
+        description={`
+          Make sure that is placed in a Datapoint is set-up to work in RTL languages. Be sure to localize \`title\`, \`value\`, \`trend.accessibilityLabel\`, and \`tooltip.accessibilityLabel\` in TileData. 
```

Comment:
This still references Datapoint. This page is for TileData.

And I still don't understand what this first sentence is telling the user? As mentioned previously, we don't provide any free-form (e.g. Node) props, so any RTL support should be handled within TileData itself. What specific things would the user need to do to support RTL?